blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/util/GlobMatcher.java

```diff
@@ -0,0 +1,33 @@
+package io.jenkins.blueocean.service.embedded.util;
+
+import org.apache.oro.text.GlobCompiler;
+import org.apache.oro.text.regex.MalformedPatternException;
+import org.apache.oro.text.regex.Pattern;
+import org.apache.oro.text.regex.Perl5Matcher;
+
+/**
+ * Matches glob string patters
+ * See https://en.wikipedia.org/wiki/Glob_(programming)
+ */
+public final class GlobMatcher {
```

Comment:
then lots of plugins will be pulling in `oro` for no reason at all. I think its fine to leave here in `util` until such time its needed elsewhere.